src/Host/Gimp/host_gimp.cpp

```diff
@@ -119,7 +119,11 @@ static GimpProcedure * gmic_qt_create_procedure(GimpPlugIn * plug_in, const gcha
 #if (GIMP_MAJOR_VERSION <= 2) && (GIMP_MINOR_VERSION <= 99) && (GIMP_MICRO_VERSION < 6)
 static GimpValueArray * gmic_qt_run(GimpProcedure * procedure, GimpRunMode run_mode, GimpImage * image, GimpDrawable * drawable, const GimpValueArray * args, gpointer run_data);
 #else
+#if (GIMP_MAJOR_VERSION <= 2) && (GIMP_MINOR_VERSION <= 99) && (GIMP_MICRO_VERSION < 19)
```

Comment:
I think all these lines should use GIMP_CHECK_VERSION() instead, e.g. here:

```
#if ! GIMP_CHECK_VERSION(2, 99, 19)
```

Reply:
Yes, that's clearer to read, will change it.